src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -152,33 +153,41 @@ public String getDisplayName() {
         return displayName;
     }
 
-    public static void configureChecker(TaskListener listener, GitHub gitHub) {
-        configureChecker(GitHubConfiguration.get().getApiRateLimitChecker(), listener, gitHub);
+    public static void configureChecker(@NonNull TaskListener listener, @NonNull GitHub gitHub) {
```

Comment:
Instead of actually checking the rate limit, we now only configure the checker to use the current listener.  Checking occurs automatically during GitHub method calls.